src/Classes/Board.js

```diff
@@ -16,11 +16,50 @@ import Mandrake from "./plants/Mandrake";
 import Player from "./Animal/Player";
 let map = document.querySelector('.map');
```

Comment:
Instead of declaring `map` as a global variable, make it a property of the `Board` class.